bot/handlers/sending_videos/compile_clips_handler.py

```diff
@@ -60,79 +65,156 @@ async def _do_handle(self, message: Message) -> None:
             return await self.__reply_no_previous_search_results(message)
 
         segments = json.loads(last_search.segments)
+        selected_segments, errors = self.__parse_segments(content[1:], segments)
 
-        selected_segments = self.__parse_segments(content[1:], segments)
+        if errors:
+            return await self._answer(message, "\n".join(errors))
```

Comment:
bez zadnej customowej wiadomosci? wyjebaly sie nastepujace rzeczy: join errors

Reply:
TODO

Reply:
dobra jak masz gdzies zapisane zeby o tym nie zapomniec to go resolve

Reply:
załatałem się nie boj